### DISCUSSION TITLE
Are MEV payouts on ETH supported?

### DISCUSSION THREAD

**Discussion by ei8ht187** [coinfoundry/miningcore#1106] [Q&A] [answered]
Hello again, 
next question from my side: Are MEV (see https://ethereum.org/en/developers/docs/mev/) supported on ETH Blockchain? If so, is it possible to set a percentage the miners get? 

Kind regards

ei8ht

**Comment by oliverw** [comment, marked as answer]
Pretty sure this is currently not supported.
